pkg/keepalive/options.go

```diff
@@ -53,6 +61,8 @@ func DefaultOption() *Options {
 		Timeout:                     10 * time.Second,
 		MaxServerConnectionAge:      Infinity,
 		MaxServerConnectionAgeGrace: 10 * time.Second,
+		WriteBufferSize:             32 * 1024,
```

Comment:
What is the gRPC default?

Reply:
default is 32k both, in google.golang.org/grpc/clientconn.go
```go
const (
        // ...
	// http2IOBufSize specifies the buffer size for sending frames.
	defaultWriteBufSize = 32 * 1024
	defaultReadBufSize  = 32 * 1024
)